### PR TITLE
fix: first sentence in step 32 JS course.

### LINKED ISSUE #48787: Requesting rewrite of first sentence in step 32 of the new JS course build an rpg
### Describe the Issue

The first sentence of step32 of the new JS course learn-basic-javascript-by-building-a-role-playing-game states the following:

`Use <code>querySelector()</code> to get the other two button elements, using their <code>#button2</code> and <code>#button3</code> <code>id</code>s.`

Attempting to translate the last half of this sentence in Arabic is a challenge because the order of the code elements cannot be changed and in Arabic the phrasing as-is would be nonsensical.
The closest I could get in my attempt would have to be something like "...using <code>#button2</code> and <code>#button3</code> which are the values of their <code>id</code> attributes."

I can do this, but the overall effect is almost to obfuscate what it is we are actually trying to say because of the extreme wordiness that I would resort to.

I, therefore, propose a slight rewording and re-organization for this sentence which is to use :

`Use <code>querySelector()</code> to get the other two <code>button</code> elements using their <code>id</code>s: <code>button2</code> and <code>button3</code>.`

### Affected Page

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62fc20387ef88d1d1998aac5.md

### Your code

```

```

### Expected behavior

My suggestion is to rewrite to this:
`Use <code>querySelector()</code> to get the other two <code>button</code> elements using their <code>id</code>s: <code>button2</code> and <code>button3</code>.`

This separates the attribute name 'id' from the values to follow and allows for easier translation.

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by WalidMoua**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #48787

rewrote the first sentence in step 32 of the new JS course build an rpg, to improve the Arabic translation

### CODE REVIEW

**Comment by raisedadead** [comment]
We are officially on Node.js 18 now. We have changed the merge requirements accordingly.

You will need to rebase this PR against the main branch OR Click the "Update branch" button to get the new checks going.

Thanks for your contribution.

**Comment by WalidMoua** [comment]
> We are officially on Node.js 18 now. We have changed the merge requirements accordingly.
> 
> You will need to rebase this PR against the main branch OR Click the "Update branch" button to get the new checks going.
> 
> Thanks for your contribution.

Thanks for the help,  I have rebased it

**Comment by Dario-DC** [comment]
I added a comment in the issue that I think should be considered before reviewing this PR

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62fc20387ef88d1d1998aac5.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-32
 
 # --description--
 
-Use `querySelector()` to get the other two button elements, using their `#button2` and `#button3` `id`s. Store them in variables called `button2` and `button3`. Remember to use `const`.
+Use `querySelector()` to get the other two `button` elements, using their `id`s:  `#button2` and `#button3`  Store them in variables called `button2` and `button3`. Remember to use `const`.
```
Comment (Sboonny) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62fc20387ef88d1d1998aac5.md:10]: Here is the new suggested string mention in the comment

```suggestion
Use `querySelector()` to get the other two `button` elements using their `id`s: `button2` and `button3`.
```

**Comment by WalidMoua** [comment]
The PR have been updated according to the new suggestion

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @WalidMoua, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Review by Sembauke — APPROVED** [review]

**Pull request merged**